### PR TITLE
Ceds 3929 remove 999l flag

### PR DESCRIPTION
**Pull request by darrylmurphy**

### CODE REVIEW

**Review by magicaljellybeans — APPROVED** [review]

File: app/controllers/navigation/Navigator.scala [hunk 1]
```diff
@@ -165,13 +163,13 @@ class Navigator @Inject() (
...
-    case AdditionalDocumentsRequired                                   => additionalDocumentsSummaryNoWaiverPreviousPage
-    case AdditionalDocumentsSummary                                    => additionalDocumentsSummaryNoWaiverPreviousPage
-    case AdditionalDocument                                            => additionalDocumentsNoWaiverPreviousPage
-    case NactCodeFirst                                                 => nactCodeFirstPreviousPage
+    case AdditionalDocumentsRequired => additionalDocumentsSummaryPreviousPage
+    case AdditionalDocumentsSummary  => additionalDocumentsSummaryPreviousPage
+    case AdditionalDocument          => additionalDocumentsPreviousPage
+    case AdditionalDocumentsRequired => additionalDocumentsSummaryNoWaiverPreviousPage
+    case AdditionalDocumentsSummary  => additionalDocumentsSummaryNoWaiverPreviousPage
+    case AdditionalDocument          => additionalDocumentsNoWaiverPreviousPage
```
Comment (udinobi) [inline app/controllers/navigation/Navigator.scala:171]: Why were these 3 lines not removed?
The compiler is also complaining (with just a warning btw) about them.

File: app/controllers/navigation/Navigator.scala [hunk 2]
```diff
@@ -260,12 +258,12 @@ class Navigator @Inject() (
...
-    case AdditionalDocument if waiver999LConfig.is999LEnabled          => additionalDocumentsPreviousPage
-    case AdditionalDocumentsRequired                                   => additionalDocumentsSummaryNoWaiverPreviousPage
-    case AdditionalDocumentsSummary                                    => additionalDocumentsSummaryNoWaiverPreviousPage
-    case AdditionalDocument                                            => additionalDocumentsNoWaiverPreviousPage
+    case AdditionalDocumentsRequired => additionalDocumentsSummaryPreviousPage
+    case AdditionalDocumentsSummary  => additionalDocumentsSummaryPreviousPage
+    case AdditionalDocument          => additionalDocumentsPreviousPage
+    case AdditionalDocumentsRequired => additionalDocumentsSummaryNoWaiverPreviousPage
+    case AdditionalDocumentsSummary  => additionalDocumentsSummaryNoWaiverPreviousPage
+    case AdditionalDocument          => additionalDocumentsNoWaiverPreviousPage
```
Comment (udinobi) [inline app/controllers/navigation/Navigator.scala:266]: Same here

File: app/controllers/navigation/Navigator.scala [hunk 3]
```diff
@@ -309,12 +307,12 @@ class Navigator @Inject() (
...
```
Comment (udinobi) [inline app/controllers/navigation/Navigator.scala:315]: Same here

File: app/controllers/navigation/Navigator.scala [hunk 4]
```diff
@@ -358,12 +356,12 @@ class Navigator @Inject() (
...
```
Comment (udinobi) [inline app/controllers/navigation/Navigator.scala:364]: Same here

**Pull request merged**